src/App.tsx

```diff
@@ -1,14 +1,24 @@
+import { useState } from 'react';
+
 import { About } from './components/About/About.tsx';
 import { Footer } from './components/footer/Footer.tsx';
 import { Header } from './components/header/Header.tsx';
+import { Products } from './components/Products/Products.tsx';
 
 import './App.css';
 
 function App() {
+    const [currentPage, setCurrentPage] = useState('Products');
```

Comment:
use a enum as a type for your useState

ALWAYS, set data type to useState generic, it is a good practise
```suggestion
    const [currentPage, setCurrentPage] = useState<string>('Products');
```